src/ocm/components/clusterDetail/ClusterDetail.tsx

```diff
@@ -72,13 +77,25 @@ const ClusterDetail: React.FC<ClusterDetailProps> = ({ cluster }) => {
               <Text component="h2">Installation progress</Text>
             </TextContent>
           </GridItem>
-          <GridItem>
+          <GridItem span={6}>
             <ClusterProgress
               cluster={cluster}
               onFetchEvents={onFetchEvents}
               totalPercentage={cluster.progress?.totalPercentage || 0}
             />
           </GridItem>
+          <GridItem span={6}></GridItem>
```

Comment:
Leftover: 
```suggestion
        
```

Reply:
Without that the progress bar itself and the alerts below that are in the same line (each one of them takes half of the screen). Do you know how can I overcome that?